app/build.gradle.kts

```diff
@@ -1,6 +1,8 @@
 plugins {
```

Comment:
@BorysChajdas  please remember that now we have "kotlin" name of packages, i see that PicoverApplication and CameraViewModel are in "java" named directory

Reply:
Yeah, rebase problem, thanks for catching it!

Reply:
Done